.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2.1
+workflows:
+  build_test:
+    jobs:
+      - build_test
+jobs:
+  build_test:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run:
+          name: Install Python Dependencies
+          command: |
+            pip install --user --no-cache-dir -r requirements.txt
```

Comment:
I think we would prefer to not add a `requirements.txt` file since we are using Pipenv. Changing this to `pip install --user --no-cache-dir black isort flake8` would be preferable I think.

@jpadilla thoughts?